namnder/internationella-namnden/body.md

```diff
@@ -1,31 +1,28 @@
 
-The Computer Science Chapter's International Group!
+## Minska Barriärerna för Internationella Studenter
 
-This is a shared group between the Computer Science Chapter and the Chapter for Media Technology, headed by International Student Coordinator Simon Olander Ålund from CS and International Organizer Simon Sundström from Media.
+Internationella nämnden har ett projekt som heter "Översätta Styrdokument". Projektet syftar till att översätta [viktiga dokument](https://styrdokument.datasektionen.se/) till engelska, så att internationella studenter kan få en bättre förståelse för vad som händer på Datasektionen. Detta är ett projekt som är öppet för alla studenter på Datasektionen. Om du är intresserad av att hjälpa till med detta, vänligen kontakta [isc@datasektionen.se](mailto:isc@datasektionen.se).
 
-The group organizes at least two events per semester for computer science students and media technology students. The group recruits new excited organizers at the beginning of each semester, but is open to new organizers all year round.
+## Evenemang
 
-By helping out in the group organizers can collect hours to count toward their 0.2 GPA bonus to the GPA points that are used to apply for international studies. The group leaders are responsible for registering the hours with THS.
+Internationella nämnden anordnar minst två evenemang per termin för internationella studenter som hör till Konglig Datasektionen eller Sektionen för Medieteknik.
+Exempel på evenemang är internationella gasquer, filmkvällar, brädspelkvällar, karaokekvällar, nätverksevenemang (för att presentera sektionens nämnderna) eller även utomhusaktiviteter som skridskoåkning eller vandring. I början av varje termin samarbetar nämnden även med datas mottagningen (bara HT) och den internationella mottagningen som anordnas av THS (varje termin). 
 
-If you are interested in joining the group as an organizer, then apply for membership in the organizers' group: [Facebook group](https://www.facebook.com/groups/2151534464869750/)
+## International Reception
 
-Also, feel free to like the group's facebook page in order to receive news of the group's activities: [Facebook page](https://www.facebook.com/METAInternationalGroup/?eid=ARDdqkLUe2uRu9PZyhqSOZSHFii_V4D0Pyur3l29_TbwBwnJdzM-PPi6HzHEvk_GjVNlp7DBRUWzpQBj)
+Internationella nämnden är inte ansvarsfull för det internationella mottagandet. THS anordnar en mottagning för internationella studenter i början av varje termin. Receptionen är ett bra tillfälle att träffa internationella studenter och delta i roliga aktiviteter. Om du vill bli en buddy och hjälpa, titta på [deras facebooksida](https://www.facebook.com/thsint).
```

Comment:
Ansvarsfull -> ansvarig
Receptionen -> mottagningen 

Ansvarsfull is more the trait of being responsible rather than being in charge.

Reception in Swedish means the reception of an office building for example.